### DISCUSSION TITLE
Change default speed setting

### DISCUSSION THREAD

**Discussion by bairhys** [supermerill/SuperSlicer#2806] [Q&A]
I have just downloaded and starting to use SuperSlicer for the first time. I cannot determine how to change the "Print Settings" -> "Speed" -> "Default Speed" setting as the setting is not enabled for editing. How do I change the "Default Speed" setting?
![default speed](https://user-images.githubusercontent.com/33058425/168456507-4c88e7a5-e9a2-4fc3-a6a7-e9c276704391.jpg)

**Comment by bairhys** [comment]
To change default speed, need to set a percentage value for a speed (e.g. Internal Perimeter Speed = 50%) which will enable the default speed setting.

I guess Default Speed sets the base value for the settings vertically below it if they use % symbol in values? 

Then the values not vertically below default speed, how are those speeds calculated when using % symbol? From the speed in the first column of the row?

![Untitled](https://user-images.githubusercontent.com/33058425/168457181-bc033bde-01d5-4417-9630-36fb758d837f.png)

**Reply by supermerill** [reply]
Yes, but you should check the tooltip, Some may be specials.
